src/pages/create-or-edit-lesson/CreateOrEditLesson.styles.ts

```diff
@@ -63,5 +63,11 @@ export const styles = {
     fontWeight: '400'
   },
   addAttachmentBtn: { width: 'fit-content' },
-  addAttachmentIcon: { ml: '5px', width: { xs: '18px', sm: '22px' } }
+  addAttachmentIcon: { ml: '5px', width: { xs: '18px', sm: '22px' } },
+  justSomeBox: {
+    width: '80%',
```

Comment:
createCourse page is wrapped by PageWrapper component, the content inside should have 100% width, do you need this 80% here?

Reply:
done